### ISSUE TITLE
[1.8.9][0-8.25] Crash when entering a RFTool Dimension

### ISSUE THREAD

**Issue by Xiaminou** [BluSunrize/ImmersiveEngineering#1122]
Client crashed when entering a freshly created RFTool Dimension from my base. Some render gone wrong?
Crash Report: http://pastebin.com/Arge9v7U

**Comment by mindforger** [comment]
my first shot would be, reproduce without Optifine :)

**Comment by Xiaminou** [comment]
I am working on it, but I don't think there's an issues with optifine in 1.8.9. I have had plenty of render bugs in 1.8.9 and none had anything to do with using optifine.

**Comment by malte0811** [comment]
Something seems to be going very wrong here. I see 2 ways how this can happen and both aren't supposed to be possible:
1. The render is called for a block of air that has a squeezer TileEntity (The TE can not be null, otherwise it would crash with a NPE a few lines above)
2. The TileEntity passed to the render is not actually in the world associated with the `TileEntityRendererDispatcher` of the TESR.
@Xiaminou I assume that you have a squeezer at your base, if not, please say so. Do you know anything about that RFTools dimension/what Dimlets did you put in there? Does it happen every time you enter that dimension? Do you crash whenever you try to log into that world/server?

**Comment by Xiaminou** [comment]
I do indeed have a squeezer in my base, moreover I have found a way to enter the RFTool Dimension by going via The End instead of my base as I figured it had something to do with the squeezer in my base and that does not crash me.
I rolled back my world after crashing the first time as it was giving me fatal error that had nothing to do with Immersive Engineering and used the method I described above.
I can now teleport to that dimension from my base.

I never retried what I did the first time because I didn't want to have to rollback my server once more, so it might have been just a single event.
